addons/auth_totp_mail_enforce/views/templates.xml

```diff
@@ -0,0 +1,27 @@
+<odoo>
+    <template id="auth_totp_mail_form" inherit_id="auth_totp.auth_totp_form">
+        <xpath expr="//form/div[1]" position="attributes">
+            <attribute name="t-if">user.totp_enabled</attribute>
+        </xpath>
+        <xpath expr="//form/div[1]" position="after">
+            <div t-else="" class="mb-2 mt-2 text-muted">
+                To login, enter below the six-digit authentication code sent to your email inbox.
+                <br/>
+            </div>
+        </xpath>
+        <xpath expr="//form[1]" position="after">
+            <form method="POST" t-if="not user.totp_enabled">
+                <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                <input type="hidden" name="send_email" value="1"/>
+                <button type="submit" class="btn btn-secondary btn-block">Re-send email</button>
+            </form>
+        </xpath>
+        <xpath expr="//div[hasclass('border-top')]" position="before">
+            <div class="mb-2" t-if="not user.totp_enabled">
+                We strongly recommend enabling the two-factor authentication using an authenticator app to help secure your account.
+                <br/>
+                <a href="https://www.odoo.com/documentation/15.0/applications/general/auth/2fa.html" title="Learn More" target="_blank">Learn More</a>
```

Comment:
This link is also included in the stable/standard `auth_totp` module
https://github.com/odoo/odoo/blob/15.0/addons/auth_totp/views/templates.xml#L12
Should I change it there also ? Despite the fact this would require a `-u` in all databases ?

Or I leave it here that way, and change this in master only, when I merge both module together ?

Besides, as you say, this mean the link to the documentation might not match the current database version, which can be confusing for the users. I would have left the link that way in my opinion.

Reply:
I think keeping the direct link to the documentation is the best option here. We have a Sphinx extension to manage redirections in the documentation so it's actually more complicated to maintain a short link than a direct one.